### ISSUE TITLE
Decouple tests from cnab-riff

### ISSUE THREAD

**Issue by scothis** [projectriff/cnab-k8s-installer-base#48]
Depending on cnab-riff for tests means that we can't make breaking changes in cnab-k8s-installer-base. This repo should define its own test bundles and run them independent of riff.

**Comment by glyn** [comment]
Agreed. Once https://github.com/projectriff/cnab-k8s-installer-base/pull/63 has landed, it would be good if this repo's CI tested relocation and that will be far more tractable with a small test bundle.

BTW it occurred to me that if we created test images on the fly and then exported  a thick bundle and then deleted the test image (docker) repositories, then if the test images are successfully loaded, that's proof they weren't loaded from their original repositories. (Need to make sure there isn't a docker daemon holding a cache of the deleted images.)
